l10n_ar_website_sale_ux/models/product_template.py

```diff
@@ -0,0 +1,54 @@
+from odoo import models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+    
+    def _get_sales_prices(self, pricelist):
+        template_price_vals = super()._get_sales_prices(pricelist)
+        if self.user_has_groups('account.group_show_line_subtotals_tax_included'):
+            partner_sudo = self.env.user.partner_id
+            fpos_id = self.env['website']._get_current_fiscal_position_id(partner_sudo)
+            fiscal_position = self.env['account.fiscal.position'].sudo().browse(fpos_id)
+
+            for product_tmpl_id, template_price_val in template_price_vals.items():
+                template = self.env['product.template'].browse(product_tmpl_id)
+                product_taxes = template.sudo().taxes_id.filtered(lambda t: t.company_id == t.env.company)
+                taxes = fiscal_position.map_tax(product_taxes)
+                template_price_vals[product_tmpl_id].update({'price_tax_excluded': taxes.with_context(force_price_include=True).compute_all(
+                                    price_unit=template_price_val['price_reduce'],
+                                    currency=pricelist.currency_id,
+                                    quantity=1,
+                                    product=template.sudo(),
+                                    partner=partner_sudo,
+                                )['total_excluded']})                
+        return template_price_vals
+
+    def _get_combination_info(self, combination=False, product_id=False, add_qty=1, pricelist=False, parent_combination=False, only_template=False):
```

Comment:
Same of above, only add that this applies to product "form" page. also you can add that price_tax_excluded key is not in the orginal result dictionary